Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/App.tsx

```diff
@@ -1,26 +1,63 @@
+import { useState } from "react";
+
 import BatteryInput from "~/components/batteryInput";
 import Header from "~/components/header";
 import SpeedInput from "~/components/speedInput";
 import WeatherInput from "~/components/weatherInput";
 
 const App = () => {
-  const calculateRange = () => {
-    return;
-  }
+  const [s, setSpeed] = useState(0);
+  const [b, setBattery] = useState(0);
+  const [w, setWeather] = useState(0);
+  const range = -((s * s * b) / 2500) + 4 * b + w;
+  const [output, setOutput] = useState<string>("");
+
+  const calculateRange = (range: any) => {
```

Comment:
When I don't provide a value for speed or battery, I only get the message `*Required Speed*`.
Also, can you add the error messages below the respective input field rather than below the submit button?